pages/_error.tsx

```diff
@@ -6,6 +6,7 @@ interface ErrorProps {
   error?: Error | null;
 }
 
+// eslint-disable-next-line sonarjs/no-globals-shadowing -- TODO not sure of reason for this
```

Comment:
@frano-m I'm not super familiar with the purpose of this file - should this stay disabled, and if so would you be able to let me know a reason to specify so I can remove the TODO? Thank you so much for taking a look at these, I know there's a lot!